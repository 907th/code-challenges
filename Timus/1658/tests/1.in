4
9 81
12 9
6 10
7 9
