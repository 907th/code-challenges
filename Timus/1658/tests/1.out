9
No solution
1122
111112
